mlir/lib/Dialect/Linalg/Transforms/TilingInterfaceImpl.cpp

```diff
@@ -120,16 +120,21 @@ struct LinalgOpTilingInterface
     Location loc = op->getLoc();
     LinalgOp linalgOp = cast<LinalgOp>(op);
     SmallVector<Value> valuesToTile = linalgOp->getOperands();
-    SmallVector<Value, 4> tiledOperands = makeTiledShapes(
+    SmallVector<Value> tiledOperands = makeTiledShapes(
         b, loc, linalgOp, valuesToTile, offsets, sizes, {}, true);
+    SmallVector<Operation *> generatedSlices = llvm::map_to_vector(
+        llvm::make_filter_range(
+            tiledOperands, [](Value v) -> bool { return v.getDefiningOp(); }),
+        [](Value v) -> Operation * { return v.getDefiningOp(); });
```

Comment:
Should these be `v.getDefiningOp<tensor::ExtractSliceOp>()` in case the operand is unaffected by tiling?

Reply:
This could be a `memref.subview` as well. I left the fusion implementation to account for what kind of operation it can index on. Ill leave it as is for now, but we can come back and revisit.

EDIT: Actually fixed it to return filter out extract slices and subview operations.